packages/ui-react/src/components/Header/Header.module.scss

```diff
@@ -27,7 +27,8 @@
   // Make header static
   //
   &.static {
-    position: static;
+    position: relative;
+    z-index: 1;
```

Comment:
Are you sure this is needed? Does the usermenu popover still work?

Reply:
Using CSS `will-change` causes layering issues. Now that the page is wrapped with the Fade component, the background image and fade cover the header. 

I tested most overlays and modals which all still look as expected.